frontend/src/components/assessment-creation/CreateAssessmentPage.tsx

```diff
@@ -0,0 +1,239 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+import React, { useState } from "react";
+import countryList from "react-select-country-list";
+import { useForm, Controller } from "react-hook-form";
+import { Select } from "chakra-react-select";
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  VStack,
+  RadioGroup,
+  Stack,
+  Radio,
+  Box,
+  Text,
+  Button,
+  FormErrorMessage,
+  Alert,
+  AlertIcon,
+} from "@chakra-ui/react";
+
+const CreateAssessmentPage = (): React.ReactElement => {
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+    control,
+  } = useForm();
+
+  type FormInputs = {
+    assessmentName: string;
+  };
+
+  const fromErrorMsg = "Please resolve all issues before publishing or saving";
+  const [validSubmit, setValidSubmit] = useState(true);
+
+  const onSubmit = (data: any, e: any) => {
+    setValidSubmit(true);
+    console.log(data, e);
+  };
+  const onError = (errs: any, e: any) => {
+    setValidSubmit(false);
+  };
+
+  const countryOptions = React.useMemo(() => countryList().getData(), []);
+  const gradeOptions = [
+    {
+      label: "K",
+      value: "k",
+    },
+    {
+      label: "Grade 1",
+      value: "1",
+    },
+    {
+      label: "Grade 2",
+      value: "2",
+    },
+    {
+      label: "Grade 3",
+      value: "3",
+    },
+    {
+      label: "Grade 4",
+      value: "4",
+    },
+    {
+      label: "Grade 5",
+      value: "5",
+    },
+    {
+      label: "Grade 6",
+      value: "6",
+    },
+    {
+      label: "Grade 7",
+      value: "7",
+    },
+    {
+      label: "Grade 8",
+      value: "8",
+    },
+  ];
+
+  return (
+    <VStack align="left" width="75%" spacing={8}>
+      <Text textStyle="eyebrow">BASIC INFORMATION</Text>
+
+      {!validSubmit && (
+        <Alert status="error">
+          <AlertIcon />
+          {fromErrorMsg}
+        </Alert>
+      )}
+
+      <FormControl isRequired isInvalid={Boolean(errors.assessmentName)}>
+        <FormLabel color="grey.400"> Assessment Name </FormLabel>
+        <Input
+          placeholder="e.g. Ontario Grade 5 Pre-Term Assessment"
+          {...register("assessmentName", {
+            required: "Please enter a name for the assessment",
+          })}
+        />
+        <FormErrorMessage> {errors.assessmentName?.message} </FormErrorMessage>
+      </FormControl>
+
+      <Box width="50%">
+        <Controller
```

Comment:
not sure if you've looked into it yet, but can you see if you are able to reuse our `SelectFormInput` component (you can make minor adjustments if needed) 

then we don't need the controller and can simplify the code a bit